fuzzers/Makefile

```diff
@@ -11,11 +13,36 @@ BITONLY=N
 all:
 
 define fuzzer
+
+# $(1) - Fuzzer name/directory
+# $(2) - Space seperated list of dependencies for the fuzzer.
+
+# Make the all target depend on the run.ok file for the fuzzer.
 all: $(1)/run.ok
+
+# Make the clean target run `make clean` in the fuzzer's directory.
 clean::
 	$$(MAKE) -C $(1) clean
+	rm -f $(1)/*.log
```

Comment:
Should be writing logs to a log directory

Reply:
The `run.ok` should probably be in the same directory as the logs?

Reply:
I'm fine with that.

Reply:
FYI - Done, will think about `run.ok` in the future...